src/plist.c

```diff
@@ -69,8 +69,8 @@ typedef volatile struct {
     int state;
 } thread_once_t;
 
-static thread_once_t init_once = {0, 0};
-static thread_once_t deinit_once = {0, 0};
+static const thread_once_t init_once = {0, 0};
```

Comment:
Are you sure this is correct? This value is used in `InterlockedExchange` (via `thread_once`), so it's not really a const, is it?